### PR TITLE
New Crowdin translations webapp_locales

### PR DESCRIPTION
**Pull request by LiteFarm-eng**
New Crowdin pull request with translations of webapp_locales

### CODE REVIEW

**Comment by kathyavini** [comment]
There were a great deal of newline errors; I have corrected on CrowdIn and will re-pull.

File: packages/webapp/public/locales/ml/translation.json [hunk 1]
```diff
@@ -748,63 +748,63 @@
...
+    "NOTES_CHAR_LIMIT": "കുറിപ്പുകളിൽ 10,000 ക്യാറക്ടറുകളിൽ കുറവായിരിക്കണം",
     "RESIDENCE": {
-      "EDIT_TITLE": "",
+      "EDIT_TITLE": "താമസസ്ഥലം എഡിറ്റ് ചെയ്യുക",
       "NAME": "വസതിയുടെ പേര്",
-      "TITLE": ""
+      "TITLE": "താമസസ്ഥലം ചേർക്കുക"
     },
     "SPOTLIGHT": {
       "ADD": "yet to be translated",
```
Comment (kathyavini) [inline packages/webapp/public/locales/ml/translation.json:779]: Oh shoot this is a holdover from demo; will let Lari know about these

File: packages/webapp/public/locales/ml/translation.json [hunk 2]
```diff
@@ -533,209 +533,209 @@
...
-    "TASK": ""
+    "TASK": "ചുമതല"
   },
   "ERROR_FALLBACK": {
-    "CONTACT": "",
-    "MAIN": "",
-    "RELOAD": "",
-    "SUBTITLE": "",
-    "TITLE": ""
+    "CONTACT": "ഇപ്പോഴും ചെളിയിൽ കുടുങ്ങിയിട്ടുണ്ടോ? വിഷമിക്കേണ്ട, നിങ്ങളെ പുറത്തെടുക്കാൻ ഞങ്ങൾ ഇവിടെയുണ്ട്:\n<1>{{supportEmail}}</1>",
```
Comment (Duncan-Brain) [inline packages/webapp/public/locales/ml/translation.json:539]: Here is another newline not present in english

File: packages/webapp/public/locales/ml/translation.json [hunk 3]
```diff
@@ -533,209 +533,209 @@
...
+      "WIDTH": "ബഫർ സോണിന്റെ വീതി"
     },
     "BULK_UPLOAD_SENSORS": {
       "DOWNLOAD_FILE": {
-        "DEFAULT": "",
-        "PARTIAL_SUCCESS_BOTTOM_TEXT": "",
-        "PARTIAL_SUCCESS_TOP_TEXT": "",
-        "ROW": "",
-        "SOME_ERRORS": ""
+        "DEFAULT": "നിങ്ങൾ ചെയ്തതിൽ എന്തെങ്കിലും തെറ്റ് സംഭവിച്ചോ?\nസഹായത്തിന് support@litefarm.org എന്നതിൽ ബന്ധപ്പെടുക.",
```
Comment (Duncan-Brain) [inline packages/webapp/public/locales/ml/translation.json:609]: And this one I don't see in english

File: packages/webapp/public/locales/ml/translation.json [hunk 4]
```diff
@@ -533,209 +533,209 @@
...
+        "PARTIAL_SUCCESS_BOTTOM_TEXT": "ഇപ്പോൾ നിങ്ങളുടെ ഫാം മാപ്പിൽ അവയെ കാണാനാകും. ഭാവിയിലെ അപ്‌ലോഡുകളിൽ ഈ സെൻസറുകൾ അവഗണിക്കപ്പെടും.",
```
Comment (Duncan-Brain) [inline packages/webapp/public/locales/ml/translation.json:610]: I think this one is missing newlines \n\n at the end.

File: packages/webapp/public/locales/ml/translation.json (hunk 3 above)
Comment (kathyavini) [reply packages/webapp/public/locales/ml/translation.json:609]: I saw that one but said "eh, it makes sense contextually" 😂  It's in a .txt file so no UI to break.

File: packages/webapp/public/locales/ml/translation.json (hunk 2 above)
Comment (kathyavini) [reply packages/webapp/public/locales/ml/translation.json:539]: Thanks, fixed.

**Review by kathyavini — APPROVED** [review]
Thanks @Duncan-Brain for your 👀s!!

Newlines should now all be fixed, `planned` has been shortened, and `yet to be translated` should all be removed from app.

Should be good to merge and test 🤞

File: packages/webapp/public/locales/ml/translation.json [hunk 5]
```diff
@@ -533,209 +533,209 @@
...
+        "FILE_ROW_LIMIT_EXCEEDED": "ഫയൽ വരിയുടെ പരിധി കവിഞ്ഞു. ഒരു ഫയലിൽ നിന്ന് അപ്‌ലോഡ് ചെയ്യാൻ കഴിയുന്ന സെൻസറുകളുടെ പരമാവധി എണ്ണം 100 ആണ്.",
+        "MISSING_COLUMNS": ". നിരകൾ ആവശ്യമാണ്/നഷ്‌ടമായി.",
+        "SENSOR_BRAND": "ഉത്പന്നതിന്റെ പേര് തെറ്റായി രേഖപെടുത്തിയിരിക്കുന്നു, 1നും 100നും ഇടയിൽ ക്യാറക്ടറുകൾ ഉണ്ടായിരിക്കണം.",
+        "SENSOR_DEPTH": "തെറ്റായ ആഴ കണക്ക്, ശരിയായ കണക്കിന് 0നും 1000നും ഇടയിൽ വരേണ്ടതാണ്.",
+        "SENSOR_HARDWARE_VERSION": "തെറ്റായ ഹാർഡ്‌വെയർ വിവരണ നാമം. ശരിയായതിൽ 1നും 100നും ഇടയിൽ ക്യാറക്ടറുകൾ ഉണ്ടായിരിക്കണം.",
+        "SENSOR_LATITUDE": "തെറ്റായ അക്ഷാംശരേഖ. -85നും 85. നും ഇടയിലും ദശാംശമുണ്ടെങ്കിൽ 10 ദശാംശത്തിൽ താഴേയും വേണ്ടതാണ്.",
+        "SENSOR_LONGITUDE": "തെറ്റായ ദേശാന്തരരേഖ. -180നും 180. നും ഇടയിലും ദശാംശമുണ്ടെങ്കിൽ 10 ദശാംശത്തിൽ താഴേയും വേണ്ടതാണ്.",
+        "SENSOR_MODEL": "തെറ്റായ മോഡൽ നാമം. ശരിയായതിൽ 1നും 100നും ഇടയിൽ ക്യാറക്ടറുകൾ വരേണ്ടതാണ്.",
+        "SENSOR_NAME": "തെറ്റായ സെൻസർ നാമം. ശരിയായതിൽ 1നും 100നും ഇടയിൽ ക്യാറക്ടറുകൾ വരേണ്ടതാണ്.",
+        "SENSOR_READING_TYPES": "തെറ്റായ വായന ഇനം (reading type) കണ്ടെത്തിയിരിക്കുന്നു: {{ reading_types }}. ശരിയായ മൂല്യങ്ങൾ ഇവയിൽ ഉൾപെടുത്തിയിരിക്കുന്നു: {{ allowed_reading_types }}."
```
Comment (Duncan-Brain) [inline packages/webapp/public/locales/ml/translation.json:650]: Is not translating the bracket here a mistake?

Comment (kathyavini) [reply packages/webapp/public/locales/ml/translation.json:650]: No idea. Maybe it's a concept that makes more sense in English? I think it's okay to leave as it doesn't mess with the interpolated values.

**Review by Duncan-Brain — APPROVED** [review]

**Pull request merged**